doc/lua_api.md

```diff
@@ -8001,9 +8001,8 @@ child will follow movement and rotation of that bone.
     * Passing no arguments resets lighting to its default values.
     * `light_definition` is a table with the following optional fields:
       * `saturation` sets the saturation (vividness; default: `1.0`).
-          values > 1 increase the saturation
-          values in [0,1) decrease the saturation
-            * This value has no effect on clients who have the "Tone Mapping" shader disabled.
+        * values > 1 increase the saturation
+        * values in [0,1] decrease the saturation
```

Comment:
Actually that was meant to read [0,1). In math ')' denotes the end of an interval *excluding* the end value.

Reply:
Please write that out, don't rely on math notation like that

Reply:
Fair enough.